config-model/src/test/java/com/yahoo/vespa/model/container/xml/JettyContainerModelBuilderTest.java

```diff
@@ -182,6 +188,38 @@ public void verify_that_old_http_config_override_inside_server_tag_works() throw
                 is(not(nullValue())));
     }
 
+    @Test
+    public void ssl_keystore_configurator_can_be_overriden() throws IOException, SAXException {
```

Comment:
This test must also verify that the KeyStoreConfigurator has been injected into the ConnectorFactory. For this purpose, use Component.getInjectedComponentIds().

Reply:
Fixed in #4159